Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+                </GridItem>
+
+                <GridItem
+                    rowStart={4}
+                    colStart={1}
+                    colSpan={2}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        bg={props.data?.motor_controller_contactor[0] ? greenBgCol : redBgCol}
+                        borderColor={borderCol}
+                        borderWidth='2px'
+                        lineHeight='1.6em'
+                        h='71.4%'
+                    >
+                        <Text as='b' fontSize='xs' color={props.data?.motor_controller_contactor[0] ? greenTxtCol : redTxtCol}>Motor {props.data?.motor_controller_contactor[0] ? 'On' : 'Off'}</Text>
+                    </Center>
+                </GridItem>
+
+                <GridItem
+                    rowStart={1}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Accelerator</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.accelerator[0].toFixed(0) ?? -1.0} {CONSTANTS.accelerator.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Accelerator Out</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.accelerator_out[0].toFixed(1) ?? -1.0} {CONSTANTS.accelerator_out.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Speed</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.speed[0].toFixed(0) ?? -1.0} {CONSTANTS.speed.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={4}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                    pl='1' 
+                    pr='1'
+                    pb='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        rangeBarH='1vh'
```

Comment:
```suggestion
                >
                    <RangeCell
                        fontSize='sm'
                        rangeBarH='1vh'
```
In order to increase the font sizes in the rows of the top middle grid with the RangeBars, we have to remove the `pb` as well